exe/cli/cmd/ffs_setdefaultcidconfig.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"os"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/textileio/powergate/ffs"
+)
+
+func init() {
+	ffsSetDefaultCidConfigCmd.Flags().StringP("token", "t", "", "FFS auth token")
+
+	ffsCmd.AddCommand(ffsSetDefaultCidConfigCmd)
+}
+
+var ffsSetDefaultCidConfigCmd = &cobra.Command{
+	Use:   "setDefaultCidConfig [(optional)file]",
```

Comment:
Nit: similar idea of merging into `pow ffs config` with `pow ffs config --default`... but looks weird since feels like pushing too much. 

Other idea is adding an extra subcommand: `pow ffs config get <cid>`, `pow ffs config prepare <cid>`, `pow ffs config default <file>`. But that might be too long?

Reply:
Yea I'd like to consolidate some things like that. Trying to figure out what feels right. I'm leaning towards something like you last idea. Also feels like maybe we can at some point flatten the entire cmd tree where it makes sense. For example, if FFS is the primary API, maybe we just move all FFS sub commands to the root command.

Reply:
Yeah, seems like `ffs` command is starting to dominate what is useful. Or maybe consider making it a separate binary.

Reply:
Ok here is what I settled on. I think it's good.

You manage configs for _upcoming_ operations using the `config` command and its sub-commands:
`pow ffs config prep [cid]` - returns a config prepped for the provided cid
`pow ffs config set [file|stdin]` - sets the default config

You interact with configs for current/existing operations using the two commands:
`pow ffs push [cid]` - push/apply a config
`pow ffs pull [cid]` - gets the current config for the cid

Reply:
`push` and `pull` can easily be moved under the `config` cmd as well, let's see what feels right.